concent_api/core/payments/backends/sci_backend.py

```diff
@@ -55,6 +57,46 @@ def get_list_of_payments(
     return payments_list
 
 
+def are_all_payment_ts_younger_than_last_payment_closure_time_if_payment_exists(
+    requestor_eth_address: str,
+    provider_eth_address: str,
+    subtask_results_accepted_list: List[ForcedPaymentEvent],
+) -> bool:
+
+    oldest_payments_ts = _get_oldest_payment_timestamp_from_subtask_results_accepted_list(subtask_results_accepted_list)
+
+    forced_payment_event_list = _get_list_of_forced_payment_events(
+        requestor_eth_address=requestor_eth_address,
+        provider_eth_address=provider_eth_address,
+        search_payments_since_ts=oldest_payments_ts,
```

Comment:
you should call with with the `youngest_payment_ts` from my PoV.

Reply:
good catch :+1: done